Assets/BossRoom/Scripts/Client/Net/ClientGameNetPortal.cs

```diff
@@ -81,7 +82,11 @@ private void OnNetworkReady()
         }
 
         private void OnSceneLoaded(Scene scene, LoadSceneMode mode)
-        {
+        {
+            if (scene.buildIndex == 3)

```

Comment:
magic number and I think we should rather compare the scene name as buildindex could change.